libs/designer-ui/src/lib/html/plugins/toolbar/DropdownColorPicker.tsx

```diff
@@ -97,7 +100,9 @@ export const DropdownColorPicker = ({
   return (
     <DropDown {...dropdownProps} disabled={disabled} stopCloseOnClickSelf buttonIconSrc={buttonIconSrc} editor={editor}>
       <div className="color-picker-wrapper" style={{ width: WIDTH }} ref={innerDivRef}>
-        <Text className="color-picker-title">{capitalizeFirstLetter(title ?? '')}</Text>
+        <Text className="color-picker-title" style={{ color: isInverted ? constants.INVERTED_TEXT_COLOR : constants.STANDARD_TEXT_COLOR }}>
```

Comment:
I'm wondering why these aren't coloring properly by default, do these html components render outside of the FluentProvider or something?

Reply:
yeah they do, which was why these weren't getting caught